### DISCUSSION TITLE
Move OpenShift into a separate repo

### DISCUSSION THREAD

**Discussion by SimonGolms** [WeblateOrg/weblate#4767] [General] [answered]
While updating the OpenShift installation in the last days (see: #4749), I noticed that this is probably not the best place where it's actually stored.
IMHO: I see the OpenShift part on the same level as the [docker-compose](https://github.com/WeblateOrg/docker-compose) or [helm](https://github.com/WeblateOrg/helm) part and find it more convenient to treat it as a separate repo.

**Comment by nijel** [comment, marked as answer]
I was thinking about this as well :-). It definitely makes sense to me to have it standalone.

**Reply by nijel** [reply]
The repo is here: https://github.com/WeblateOrg/openshift

The docs and Weblate repo still need updates, I will do it once everything is working in the new location.

**Comment by comradekingu** [comment]
@SimonGolms +1
Your heraldic lion is missing the penis btw, which is not honorable symbolism. Usually given to cowardly units.
I only care that you know :)

**Comment by orangesunny** [comment]
@comradekingu Weblate is libre software; everyone can use it, every kind person is invited, and we appreciate that. Every honest and well-meant commit or comment is welcomed. Let’s keep it that way and not insult each other. Thank you for respecting that. We honor humor, but that was ruder than funny.

@SimonGolms Thank you for the idea, let’s keep things mooving!

**Reply by comradekingu** [reply]
I didn't make that up. Nor is the information an insult of any kind…

**Reply by orangesunny** [reply]
Well, 
- to keep the ball rolling: You never know what others can see from their points of view.
- To talk about animal body parts, there are way better places than Weblate repos and discussions. After all, Weblate is not specialized in the fields of zoology and wildlife. But there are better plaftorms to do so, like https://www.instagram.com/davidattenborough/ Not so libre, but really professional. And on topic 🤓 
- Let’s move on.

**Reply by comradekingu** [reply]
>to keep the ball rolling: You never know what others can see from their points of view.

Except when it is:

"_This comment was marked as disruptive content_. "

Edit: While I am not suggesting Weblate should be knowledgeable in zoology or wildlife, I am actually not making a point about either.

**Reply by SimonGolms** [reply]
@orangesunny: thank you for taking the floor and handling the situation, where I agree with you on all points. Feel free to mark this comment also as off-topic.

@comradekingu: fyi - my current profile picture shows the emblem of my hometown and represents a royal beaver. You could have recognized it by its tail.
